addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3120,28 +3192,54 @@ options.registry.WebsiteAnimate = options.Class.extend({
     /**
      * @private
      */
-    _forceAnimation() {
-        const $scrollingElement = $().getScrollingElement();
+    async _forceAnimation() {
         this.$target.css('animation-name', 'dummy');
-        // Trigger a DOM reflow.
-        void this.$target[0].offsetWidth;
-        this.$target.addClass('o_animating');
-        $scrollingElement[0].classList.add('o_wanim_overflow_xy_hidden');
-        this.$target.css('animation-name', '');
-        this.$target.one('webkitAnimationEnd oanimationend msAnimationEnd animationend', () => {
-            $scrollingElement[0].classList.remove('o_wanim_overflow_xy_hidden');
-            this.$target.removeClass('o_animating');
-        });
+
+        if (this.$target[0].classList.contains('o_animate_on_scroll')) {
+            // Trigger a DOM reflow.
+            void this.$target[0].offsetWidth;
+            this.$target.css('animation-name', '');
+            this.$window.trigger('resize');
+        } else {
+            // Trigger a DOM reflow.
+            await new Promise(resolve => {
+                setTimeout(resolve);
+            });
+            this.$target.addClass('o_animating');
+            this.trigger_up('cover_update', {
+                overlayVisible: true,
+            });
```

Comment:
Becasue of the change in `toggleOverlayVisibility`... It is needed to hide the overlay when animating

Reply:
Even the Promise+setTimout? If so, use `concurrency.delay()` or write it in one line: `await new Promise(resolve => setTimeout(resolve))`, a bit prettier :+1:

Reply:
The Promise+setTimout has been added to prevent the animation from being launched twice when previewing the new "Intensity" option. I'll change it to just one line :+1: